packages/default/scss/forms/_layout.scss

```diff
@@ -379,24 +379,6 @@
     }
 
 
-    // Edit buttons
-    .k-edit-buttons {
-        padding: $toolbar-padding-y $toolbar-padding-x;
-        border-width: 1px 0 0;
-        border-style: solid;
-        border-color: inherit;
-        text-align: right;
-        clear: both;
-    }
-    .k-edit-buttons .k-button + .k-button {
-        margin-left: ( $spacer-x / 2 );
-    }
-    .k-edit-form-container .k-edit-buttons {
-        margin: $window-inner-padding-y (-$window-inner-padding-x) (-$window-inner-padding-y);
-        padding: $window-inner-padding-y $window-inner-padding-x;
-    }
-
-
```

Comment:
The `.k-edit-buttons` is currently used in Kendo jQuery Grid, TreeList, Gant. Scheduler and Editor components and this change also requires changing the markup in jQuery. I currently see that the Grid and Editor tests in the themes are updated, however, please make sure that the other component tests are also updated.

I also believe we should also log and link a separate issue for updating the rendering in jQuery.